onboarding-enabler-java/src/test/java/org/zowe/apiml/eurekaservice/client/impl/ApiMediationClientImplTest.java

```diff
@@ -47,6 +48,7 @@ public void startEurekaClient() throws ServiceDefinitionException {
         ApiMediationServiceConfig config = new ApiMediationServiceConfig();
         config.setApiInfo(Collections.singletonList(apiInfo));
         config.setCatalog(catalogUiTile);
+        config.setAuthentication(authentication);
         config.setRoutes(routes);
         config.setDescription("Example for exposing a Spring REST API");
         config.setTitle("Hello Spring REST API");
```

Comment:
Why do we use the setter pattern instead of providing the information in the constructor?

Reply:
Honestly, I don't know why it was done like this by Pavlin. I didn't want to do big changes. Maybe this is a good candidate for refactoring.

Reply:
If there are more parameter, builder pattern would be good.